### ISSUE TITLE
BUG: can't compile linux binary Fedora

### ISSUE THREAD

**Issue by timofeika** [ezQuake/ezquake-source#936]
In main page in description how to build linux binary for Fedora, not enough one package in dependencies list, in build-linux.sh it's exist, "minizip-devel".

**Comment by dsvensson** [comment]
I'm not well versed with Fedora, but looks like it's packaged, https://packages.fedoraproject.org/pkgs/minizip/minizip-devel/

Alternatively as you seem to be building from a git checkout, you could build a static binary instead which will not rely on distribution dependencies. Make sure that cmake, ninja, autoconf automake, libtool and pkg-config are installed and run:
```
./bootstrap.sh
cmake --preset static
cmake --build build-static --config Release
```
It will take a couple of minutes on a modern machine as it builds all the deps. It's newly introduced so hopefully it works as great on Fedora as elsewhere.

**Comment by dsvensson** [comment]
...and an even easier solution if you don't have any code changes is downloading an AppImage:

https://builds.quakeworld.nu/ezquake/releases/latest/linux/x86_64/
https://builds.quakeworld.nu/ezquake/snapshots/latest/linux/x86_64/

**Comment by VVD** [comment]
Issue is "Fix README - add minizip-devel for Fedora build section". :-D

**Comment by dsvensson** [comment]
@VVD No, not it, it's listed here, https://github.com/QW-Group/ezquake-source/blob/master/build-linux.sh#L13 

perhaps Fedora has different repos and not all enabled by default.

**Comment by dsvensson** [comment]
Something with the @timofeika's Fedora installation at least. I just tried it in Docker here,

```
docker run  --rm -it -v `pwd`:/src fedora bash
```

I had to run `yum update` and installing `git` before running `./build-linux.sh` but after that it worked as expected.

**Comment by VVD** [comment]
> @VVD No, not it, it's listed here, https://github.com/QW-Group/ezquake-source/blob/master/build-linux.sh#L13
> 
> perhaps Fedora has different repos and not all enabled by default.

Yes. We discuss this in Discord before @timofeika create this issue.
Check README: https://github.com/QW-Group/ezquake-source/blob/master/README.md?plain=1#L154
Script have minizip-devel, but README - no.

**Comment by timofeika** [comment]
Indeed, @VVD is right :P

**Comment by dsvensson** [comment]
Yes that whole readme needs some shaping up. I'm doing it iteratively as PRs merge. It's very outdated. macOS part has been updated, and there's a PR that contains update for the Windows build. Linux/BSD build coming up next.
